drivers/claims_reporting/app/models/claims_reporting/medical_claim.rb

```diff
@@ -285,7 +285,27 @@ def surgical_procedure_codes
 
     def procedure_with_modifiers
       # sort is here since this is used as a key to match against other data
-      ([procedure_code] + modifiers.sort).join('>').to_s
+      # There are a few cases where we treat a modifier as part of a procedure code, so we address them here
+      pc = procedure_code
+      mods = modifiers
+      case pc
+      when 'G9007'
+        if mods.include?('U5')
+          pc = 'G9007>U5'
+          mods.delete('U5')
+        end
+      when 'T1023'
+        if mods.include?('U6')
+          pc = 'T1023>U6'
+          mods.delete('U6')
+        end
+      when 'T2024'
+        if mods.include?('U4')
+          pc = 'T2024>U4'
+          mods.delete('U4')
+        end
+      end
+      ([pc] + mods.sort).join('>').to_s
```

Comment:
I think it would be safer in the long run to:
```
pc, *ex = procedure_code.split('>')
mods + ex
([pc] + mods.uniq.sort).join('>').to_s
```

ex:
```
procedure_code = 'G9007>U5>U4'
mods = ['U5', 'U2']
```
would result in
```
"G9007>U2>U5"
```

Reply:
But maybe I'm missing the issue.  At some future point we are comparing to this string and want U5 to go first in the above example.

Reply:
Yes -- it appears in the QA as 'G9007>U5>U2', so we need to match the order that it appears there.